Sources/MongoSwift/MongoError.swift

```diff
@@ -522,7 +522,7 @@ internal func toErrorString(_ error: bson_error_t) -> String {
     }
 }
 
-internal let failedToRetrieveCursorMessage = "Couldn't get cursor from the server"
+internal let failedToRetrieveCursorMessage = "Expected libmongoc to return a cursor, unexpectedly got nil"
```

Comment:
this seemed inaccurate, since if this happens it's not necessarily a server issue and is more likely a bug in libmongoc, which is not supposed to ever return nil from the corresponding methods